shortfin/CMakeLists.txt

```diff
@@ -256,7 +319,7 @@ if(SHORTFIN_BUILD_TESTS)
   enable_testing()
 endif()
 
-
+add_custom_target(shortfin_testdata_deps)
```

Comment:
Should this be moved under the `if(SHORTFIN_BUILD_TESTS)` above?

Reply:
No - various things expect the target to exist even if we don't actually enable testing.

Reply:
Fixed.